CHANGELOG.md

```diff
@@ -28,6 +29,231 @@
+  [#4528](https://github.com/Kong/kong/pull/4528)
+- A new core entity, "CA Certificates" has been introduced and can be accessed
+  via the new `/ca_certificates` Admin API endpoint. CA Certificates entities
+  will be used as CA trust store by Kong. Certificates stored by this entity
+  need not include their private key.
+  [#4798](https://github.com/Kong/kong/pull/4798)
+- Healthchecks now use the combination of IP + Port + Hostname when storing
+  upstream health information. Previously, only IP + Port were used. This means
+  that different virtual hosts served behind the same IP/port will be treated
+  differently with regards to their health status. New endpoints were added to
+  the Admin API to manually set a Target's health status.
+  [#4792](https://github.com/Kong/kong/pull/4792)
+
+##### Configuration
+
+- :fireworks: A new section in the `kong.conf` file describes [injected Nginx
+  directives](https://konghq.com/blog/kong-ce-nginx-injected-directives/)
+  (added to Kong 0.14.0) and specifies a few default ones.
+  In future releases, we will gradually increase support for injected Nginx
+  directives. We have high hopes that this will remove the occasional need for
+  custom Nginx configuration templates.
+  [#4382](https://github.com/Kong/kong/pull/4382)
+- :fireworks: New configuration properties allow for controling the behavior of
+  upstream keepalive connections. `nginx_http_upstream_keepalive_requests` and
+  `nginx_http_upstream_keepalive_timeout` respectively control the maximum
+  number of proxied requests and idle timeout of an upstream connection.
+  [#4382](https://github.com/Kong/kong/pull/4382)
+- New flags have been added to the `*_listen` properties: `deferred`, `bind`,
+  and `reuseport`.
+  [#4692](https://github.com/Kong/kong/pull/4692)
+
+##### CLI
+
+- :fireworks: **Database export** via the new `kong config db_export` CLI
+  command. This command will export the configuration present in the database
+  Kong is connected to (Postgres or Cassandra) as a YAML file following Kong's
+  declarative configuration syntax. This file can thus be imported later on
+  in a DB-less Kong node or in another database via `kong config db_import`.
+  [#4809](https://github.com/Kong/kong/pull/4809)
+
+##### Admin API
+
+- Many endpoints now support more levels of nesting for ease of access.
+  For example: `/services/:services/routes/:routes` is now a valid API
+  endpoint.
+  [#4713](https://github.com/Kong/kong/pull/4713)
+- The API now accepts `form-urlencoded` payloads with deeply nested data
+  structures. Previously, it was only possible to send such data structures
+  via JSON payloads.
+  [#4768](https://github.com/Kong/kong/pull/4768)
+
+##### Plugins
+
+- :fireworks: **Newly open-sourced plugin**: the [session
+  plugin](https://github.com/Kong/kong-plugin-session) (previously only
+  available in Enterprise) is now bundled in Kong. It can be used to manage
```

Comment:
minor note: this plugin has always been open sourced. It was bundled with Kong Enterprise previously, and this is now the first release of Kong that has it bundled. It differs from enterprise in that it fully utilizes the pdk and has upgraded code to work with >=1.2 such as removing baseplugin inheritance

Reply:
@darrenjennings Thanks for pointing that out! Just pushed the change to the branch.